src/tabs/TabsV2.tsx

```diff
@@ -0,0 +1,254 @@
+        const className = tab.id === selectedTabId ? ClassNames.TABACTIVE : ClassNames.TABINACTIVE;
+        return (
+            <li key={index} role="presentation" className={ClassNames.TABITEM}>
+                <Button
+                    id={`tab-${tab.id}`}
+                    aria-controls={`panel-${tab.id}`}
+                    aria-selected={tab.isSelected}
+                    className={className}
+                    disabled={tab.isDisabled}
+                    onClick={evt => this.normalTabClick(evt, tab)}
+                >
+                    {tab.name}
+                </Button>
+            </li>
+        );
+    };
+
+    //Render Tab bar
+    private renderTabLinks = () => {
+        const {id, tabOrientation, overflowTabsFrom} = this.props;
+        const {tabsData} = this.state;
+        let isOverflowRendered = false;
+        return (
+            <ul className={ClassNames.TABMAIN} role="tablist" id={id}>
+                {tabsData.map((tab: TabV2Details, index: number) => {
+                    tabsData;
+                    if (tabOrientation === TabV2Orientation.HORIZONTAL && index === overflowTabsFrom) {
+                        isOverflowRendered = true;
+                        return this.renderOverflowTab(tabsData.slice(index, tabsData.length), index);
+                    }
+
+                    //Render normal tab unless overflow tab rendered
+                    if (!isOverflowRendered) {
+                        return this.renderTabLink(tab, index);
+                    }
+                })}
+            </ul>
+        );
+    };
+
+    //Render Overflow Tab
+    private renderOverflowTab = (overflowTabs: TabV2Details[], index: number) => {
+        return (
+            <li className={ClassNames.TABITEM} key={index}>
+                <Dropdown
+                    showCaret={false}
+                    button={{
+                        icon: {shape: "ellipsis-horizontal"},
+                        link: true,
+                        className: ClassNames.TABINACTIVE,
+                    }}
+                >
+                    <DropdownMenu>
+                        {overflowTabs.map((tab: TabV2Details, index: number) => {
+                            return (
+                                <DropdownItem
+                                    key={index.toString()}
+                                    menuItemType={MenuItemType.ITEM}
+                                    isHeaderChild={true}
+                                    label={tab.name}
+                                    onClick={evt => this.overflowTabClicked(evt, tab)}
+                                    isDisabled={tab.isDisabled}
+                                />
+                            );
+                        })}
+                    </DropdownMenu>
+                </Dropdown>
+            </li>
+        );
+    };
+
+    private static renderTabPane(children: utils.ReactChildren, id: string): utils.ReactChildren {
```

Comment:
any particular reason to make this method static ?

Reply:
Was trying to utilize this func in TabPane so made static. but that didnt work so changing this func to normal private func. Thanks for the observation @akate1 .